modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -105,6 +101,15 @@
 			height: var( --n-tabs-title-height );
 			width: var( --n-tabs-title-width );
 
+			&:not( .e-active ) {
+				background-color: var( --n-tabs-title-background-color );
+
+				&,
+				& a {
```

Comment:
I realised that the tab title text element gets an `a href`-element inside the front end, which can cause clashes with the default link styling. 

![image](https://user-images.githubusercontent.com/94352322/206871602-34069272-c41e-4b48-a738-2611650ee8ca.png)

Reply:
The appears when the `Accessibility Improvements` experiment is inactive.
Good catch!!